src/AppRoutes.jsx

```diff
@@ -5,19 +5,25 @@ import Login from './components/auth/Login.jsx';
 import Register from './components/auth/Register.jsx';
 import SearchBar from './components/search/SearchBar.jsx';
 import PaymentForm from './components/payment/PaymentForm.jsx';
+import Footer from './components/layouts/Footer.jsx'
 
 
 const AppRoutes = () => {
   return (
     <BrowserRouter>
+      <div className="flex flex-col min-h-screen">
```

Comment:
I'm not sure we need another container around semantic tags

Reply:
I think it is for footer positioning. We need  them: 1 - flex (the main container)
2.  main container has flex-grow to push out footer to the bottom. In this case even if main container is empty the footer will be in the right position

Reply:
Please format AppRotes.jsx